themes/navy/layout/press-kit-status-press.ejs

```diff
@@ -0,0 +1,78 @@
1799 11.1746 22.6879 11.1746C23.1535 11.1746 23.5767 11.5555 23.5767 12.0212H22.6879H23.5767C23.5767 12.4868 23.5344 12.91 23.4921 13.3334C23.4074 13.799 23.0265 14.1376 22.6031 14.1376ZM1.48148 12.9524C1.01587 12.9524 0.59259 12.5714 0.59259 12.1058C0.59259 11.6825 0.634918 11.2593 0.677246 10.8359C0.719573 10.3703 1.14286 10.0317 1.65079 10.0741C2.1164 10.1164 2.49735 10.5397 2.4127 11.0476C2.37037 11.3863 2.37037 11.7672 2.37037 12.1058C2.32804 12.5291 1.94709 12.9524 1.48148 12.9524ZM22.2223 9.9471C21.8413 9.9471 21.5027 9.69312 21.3757 9.31216C21.291 8.97355 21.164 8.63493 20.9947 8.29632C20.8254 7.83065 21.037 7.32277 21.4603 7.15343C21.9259 6.98412 22.4339 7.19578 22.6031 7.61903C22.7725 7.99999 22.8995 8.42331 23.0265 8.84656C23.1535 9.31216 22.8995 9.77776 22.4339 9.9471C22.3915 9.9471 22.3069 9.9471 22.2223 9.9471ZM2.24338 8.88891C2.1164 8.88891 2.03174 8.88891 1.90476 8.8042C1.43915 8.63493 1.22751 8.12698 1.43915 7.66138C1.60846 7.28042 1.77778 6.85714 1.98941 6.51852C2.24338 6.09523 2.75132 5.92592 3.1746 6.17989C3.59788 6.43386 3.76719 6.9418 3.51322 7.36505C3.34391 7.70373 3.1746 7.99999 3.04762 8.3386C2.92063 8.67722 2.58201 8.88891 2.24338 8.88891ZM20.3174 6.30688C20.0635 6.30688 19.8095 6.17989 19.6402 5.96825C19.4286 5.67196 19.1746 5.41799 18.9206 5.16402C18.582 4.82539 18.582 4.27513 18.9206 3.9365C19.2592 3.59788 19.8095 3.59788 20.1482 3.9365C20.4444 4.2328 20.7407 4.57142 20.9947 4.91005C21.291 5.291 21.2487 5.84126 20.8677 6.13757C20.6984 6.22222 20.5291 6.30688 20.3174 6.30688ZM4.5291 5.46031C4.31746 5.46031 4.06349 5.37566 3.89418 5.20634C3.55555 4.86773 3.55555 4.31745 3.89418 3.97884C4.19047 3.68253 4.5291 3.38624 4.86773 3.13227C5.24868 2.83598 5.79894 2.8783 6.09523 3.25926C6.39154 3.64021 6.3492 4.19047 5.96825 4.48677C5.67196 4.69841 5.41799 4.95237 5.16402 5.20634C4.99471 5.37566 4.78307 5.46031 4.5291 5.46031ZM17.1429 3.64021C17.0159 3.64021 16.8465 3.59788 16.7195 3.51322C16.3809 3.34391 16.0847 3.1746 15.746 3.04761C15.2804 2.8783 15.0688 2.37037 15.2804 1.90476C15.4497 1.43915 15.9577 1.22751 16.4233 1.43915C16.8043 1.60846 17.2275 1.77778 17.5661 1.98941C17.9894 2.20106 18.1587 2.75132 17.9048 3.1746C17.7778 3.4709 17.4815 3.64021 17.1429 3.64021ZM7.99999 3.13227C7.66138 3.13227 7.32277 2.92063 7.19578 2.58201C7.02646 2.1164 7.23806 1.60846 7.66138 1.43915C8.04235 1.26984 8.46559 1.14286 8.88891 1.01587C9.35451 0.888886 9.82011 1.14286 9.98938 1.60846C10.1587 2.07407 9.86246 2.53968 9.39679 2.70899C9.05818 2.79365 8.71957 2.92063 8.38096 3.08995C8.21162 3.13227 8.08463 3.13227 7.99999 3.13227ZM13.2064 2.41269C13.164 2.41269 13.164 2.41269 13.1217 2.41269C12.7831 2.37037 12.4021 2.37037 12.0635 2.37037C11.5979 2.37037 11.1746 1.98941 11.1746 1.48148C11.1746 1.01587 11.5555 0.59259 12.0212 0.59259C12.0212 0.59259 12.0635 0.59259 12.1058 0.59259C12.4868 0.59259 12.91 0.634918 13.3334 0.677246C13.799 0.719573 14.1799 1.14286 14.0952 1.65079C14.0529 2.07407 13.672 2.41269 13.2064 2.41269Z" fill="white"/> </svg>		
```

Comment:
Can we extract the SVGs into files? This doesn't make for very readable HTML template.

Reply:
I've replaced 10 svgs with images and also removed package-lock.json